apps/frontend/src/pages/projects/components/project-card/project-card.tsx

```diff
@@ -1,13 +1,23 @@
-import { type ProjectGetAllItemResponseDto } from "~/modules/projects/projects.js";
+import { NavLink } from "react-router-dom";
+
+import { AppRoute } from "~/libs/enums/enums.js";
+import { type ProjectResponseDto } from "~/modules/projects/projects.js";
 
 import styles from "./styles.module.css";
 
 type Properties = {
-	project: ProjectGetAllItemResponseDto;
+	project: ProjectResponseDto;
```

Comment:
we don't need ProjectResponseDto
```suggestion
	project: ProjectGetAllItemResponseDto;
```